refined/laws/shared/src/main/scala/kantan/codecs/refined/laws/discipline/arbitrary.scala

```diff
@@ -31,6 +31,8 @@ trait ArbitraryInstances
 
   implicit val arbLegalPositiveIntString: Arbitrary[LegalString[Int Refined Positive]] =
     arbLegalValue(_.toString)
+  // format: off
```

Comment:
Why are you disabling formatting here?

Reply:
`CodecValue.IllegalValue((-math.abs(i)).toString)))` becomes `CodecValue.IllegalValue(-math.abs(i).toString)))`

Reply:
Huh. A bug in the formatter. That's a little surprising.